### ISSUE TITLE
Standard library not found

### ISSUE THREAD

**Issue by martinfinke** [clemos/haxe-sublime-bundle#240]
When I use the bundle on a file that's not in a subdirectory (with `package;`), the status bar reads:

> Main (cpp:Build)

But when I use it with a file in a subdirectory (with `package foo;`), it reads:

> Main (cpp:Build), Standard library not found

My *Haxe.sublime-settings*:

```json
{
    "haxe_path": "/usr/local/haxe/haxe",
    "haxe_library_path": "/usr/local/haxe/std",
    "haxelib_path": "/usr/local/haxe/haxelib"
}
```

The `haxe` and `haxelib` locations are correct. In `/usr/local/haxe/std` there are *Array.hx*, *Class.hx*, etc.

I'm using the latest version from Package Control with Haxe 3.2.0 and ST3 3083.

**Comment by r3a1ay** [comment]
Try to use this setting:
```json
"haxe_library_path": "/usr/local/haxe/std:."
```

**Comment by r3a1ay** [comment]
Try the latest version.
Close subleme text and kill all haxe processes.

**Comment by r3a1ay** [comment]
Should be fixed now. Please confirm.

**Comment by martinfinke** [comment]
Yes the message is ok now. However, when I use Cmd+B to run the *Hxml* build system, the console shows the message "Standard library not found". This happens for files with `package;` as well as for files in subdirectories.

**Comment by r3a1ay** [comment]
Hm, strange.
Did you kill haxe process/restart sublime after the update?

**Comment by clemos** [comment]
I believe Cmd+B will use ST's "basic" Build System, which won't use these haxe specific settings.
This is due to MacOS not "sharing" environment variables properly, and has been solved in this issue : 
https://github.com/clemos/haxe-sublime-bundle/issues/216
